test/Parlot.Tests/ScannerTests.cs

```diff
@@ -44,6 +44,19 @@ public void ShouldReadStringWithEscapes(string text, string expected)
             Assert.Equal(expected, result.GetText());
         }
 
+        [Theory]
+        [InlineData("'Lorem ipsum'", "'Lorem ipsum'")]
+        [InlineData("'Lorem \n ipsum'", "'Lorem \n ipsum'")]
+        [InlineData("'Lorem '' ipsum'", "'Lorem '' ipsum'")]
+        [InlineData(@"""Lorem """" ipsum""", "\"Lorem \"\" ipsum\"")]
+        public void ShouldReadNonEscapableString(string text, string expected)
```

Comment:
Missing some tests that shows when it's invalid.

Reply:
Also when 2,3, more quotes are used. Or when more groups of n quotes are used. 
Also when start and end char are not the same (though I mention I don't see the point in another comment)

Reply:
when what is invalid ? I don't get it ?

Reply:
All the year are positive. You need to add some to show when it's not valid and Fails() is called. To be sure that it doesn't "match" when it shouldn't.